src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx

```diff
@@ -93,15 +92,42 @@ module.exports = function ({ match }) {
         </div>
         <Button
           size={ButtonSizes.MEDIUM}
-          className={`${button} ${data.isInstalled ? disabledButtonOverride : ''}`}
-          disabled={data.isInstalled}
-          onClick={() => {
+          className={`${button} ${data.isInstalled ? ButtonColors.RED : ''}`}
+          disabled={data.isInstalling}
+          onClick={async () => {
+            if (data.isInstalling) {
+              return;
+            }
             if (!data.isInstalled) {
-              cloneRepo(data.url, powercord, data.type).then(fetchInfo);
+              setData({
+                ...data,
+                isInstalling: true
+              });
+              // Give React time to update the button since the clone command causes the client to freeze for a sec
+              setTimeout(async () => {
```

Comment:
hmmm, I don't like this, I think it would be better to have a useEffect which runs the cloneRepo function once isInstalling is set to true